libraries/AP_HAL_Linux/Storage.h

```diff
@@ -2,7 +2,8 @@
 #define __AP_HAL_LINUX_STORAGE_H__
 
 #if CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_ERLEBOARD || CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_PXF
-#define LINUX_STORAGE_USE_FRAM 1
+#define LINUX_STORAGE_USE_FRAM 0
```

Comment:
What's this? Yesterday I was using @tridge's pxf board and it was complaining about not being able to read FRAM... Is this something we should have a PR for?